src/main/java/org/folio/service/transactions/TemporaryTransactionService.java

```diff
@@ -26,7 +26,8 @@ public Future<List<Transaction>> getTransactions(LedgerFiscalYearRolloverBudget
       .withOperation("AND")
       .withJson("fromFundId", "=", budget.getFundId());
 
-    return temporaryTransactionDAO.getTempTransactions(criterionBuilder.build(), client);
+    return client.getPgClient()
+      .withTrans(conn -> temporaryTransactionDAO.getTempTransactions(criterionBuilder.build(), conn));
```

Comment:
I think we can use here just this method: .withConn() , because transaction is not needed here

Reply:
Done